examples/api/csharp/rcs/Rcs-Api-Console-Tester/RcsConsoleApp/Program.cs

```diff
@@ -23,107 +23,118 @@ static async Task Main(string[] args)
 			#region create_client
 
 			//Directory to IDEA StatiCa installation on your computer.
-			string directoryPath = "C:\\Program Files\\IDEA StatiCa\\StatiCa 23.1";
-
-			//Pass path to a new RCS Client Factory
-			using (var rcsClientFactory = new RcsClientFactory(directoryPath))
+			string directoryPath = "C:\\Program Files\\IDEA StatiCa\\StatiCa 23.1\\net6.0-windows";
```

Comment:
I didn't think you needed the 'net6.0windows' ?? why are we making the user specify this.

Reply:
I am not sure - the logic was implemented by Tomas. Personally I would get setup from windows registry - same as Python.
The implementation for dotnet is in [IdeaStatiCa.PluginsTools.Windows](https://github.com/idea-statica/ideastatica-public/blob/590088ed1c6b8d1d4662eff20604f8f2a4983d17/src/IdeaStatiCa.PluginsTools.Windows/Utilities/IdeaStatiCaSetupTools.cs#L23)